src/components/Forms/CustomSelect/index.tsx

```diff
@@ -1,32 +1,55 @@
 import React, { useState } from "react";
 import { CustomLabel } from "../CustomLabel";
+import { OutsideAlerter } from "../../../containers/OutsideAlerter";
 
 interface CustomSelectProps {
     id: string,
     placeholder: string,
     theme: "as-button" | "as-field"
+    buttonClass?: string,
     label?: string
 }
 
-export const CustomSelect:React.FunctionComponent<CustomSelectProps> = ({ id, placeholder, theme, label, children }) => {
+export const CustomSelect:React.FunctionComponent<CustomSelectProps> = (
+    {
+        id,
+        placeholder,
+        theme,
+        label,
+        buttonClass = "",
+        children
+    }) => {
     const [isDropdownOpen, setDropdownOpen] = useState(false)
     const toggleDropdown = () => setDropdownOpen(!isDropdownOpen)
+    const closeDropdown = () => setDropdownOpen(false)
+
+    // could you please advise smth better than this approach?
```

Comment:
If you're looking for click outside listener, you may have a look at [this one](https://stackoverflow.com/a/60657055/11299053) (despite implemented for class-based component, I'm sure, you can implement that logic with function component and `useEffect()`-hook, or better yet, make this feature re-usable (e.g. with custom hook).